lib/components/artist/biography.js

```diff
@@ -21,7 +21,7 @@ class Biography extends React.Component {
 
   render() {
     const artist = this.props.artist;
-    if (artist.blurb.length === 0 && artist.bio.length === 0) { return null; }
+    if (!artist.blurb && !artist.bio) { return null; }
```

Comment:
[Confirmed, nope](http://stackoverflow.com/questions/32139078/null-safe-property-access-in-es6-es2015)